fe/fe-core/src/main/java/com/starrocks/StarRocksFE.java

```diff
@@ -105,7 +105,9 @@ public static void start(String starRocksDir, String pidDir, String[] args) {
             }
 
             // init config
-            new Config().init(starRocksDir + "/conf/fe.conf");
+            Config config = new Config();
```

Comment:
I think you can use the `fe.conf` directly. You can add the entry into `fe.conf` directly.

Reply:
I don't think it is good to write back to `fe.conf` for a few reasons.

1. Despite the complexity, it is weird that fe.conf has been modified by the process itself, it will be hard to track the original configuration and confusing the operators who runs the services.
2. In a mature production deployment, the `fe.conf` is most likely managed by auto-deploy tools, any change to the fe.conf will be reverted back to its original state, unless it is modified from the source and rolled out to the production. This will make the feature useless.